src/System.Memory/src/System/SpanExtensions.Fast.cs

```diff
@@ -47,6 +47,20 @@ public static partial class SpanExtensions
         /// <exception cref="System.ArgumentNullException">Thrown when <paramref name="text"/> is null.</exception>
         public static ReadOnlySpan<char> AsReadOnlySpan(this string text) => Span.AsReadOnlySpan(text);
 
+        /// <summary>Creates a new <see cref="ReadOnlyMemory{T}"/> over the portion of the target string.</summary>
+        /// <param name="text">The target string.</param>
+        /// <exception cref="System.ArgumentNullException">Thrown when <paramref name="text"/> is a null reference (Nothing in Visual Basic).</exception>
+        public static ReadOnlyMemory<char> AsReadOnlyMemory(this string text) => Span.AsReadOnlyMemory(text);
```

Comment:
You don't need these if you add the extension methods to SpanExtensions.cs (which contains the subset of extension methods that have common implementations between the what's in Private.CoreLib and what's in System.Memory).
https://github.com/dotnet/corefx/blob/master/src/System.Memory/src/System.Memory.csproj#L25

Reply:
I don't understand.  How could this be implemented in SpanExtensions.cs when it would need access to the internal ctor and GetObjectStartLength methods in the span in corelib?

Reply:
Ah yes. Good point. It slipped my mind that the implementation requires access to internal methods.